web-ng/src/app/services/project/project.service.ts

```diff
@@ -67,4 +67,8 @@ export class ProjectService {
   updateRole(projectId: string, email: string, role: Role): Promise<void> {
     return this.dataStore.updateRole(projectId, email, role);
   }
+
+  createProject(projectId: string) {
+    return this.dataStore.updateProjectTitle(projectId, '');
```

Comment:
It's not clear from local context why we'd be calling `updateProjectTitle` here as opposed to something more general or specific. Also, if we're creating a new project, I assume we wouldn't have a projectId yet, so this would be no-arg.

Could we add a no-arg method to `DataStore`, something like `createEmptyProject()` that returns a Promise w/the new project id?